src/main/api.ts

```diff
@@ -66,6 +69,18 @@ export default {
     const { result } = await ipc_showOpenDialog.renderer!.trigger(options);
     return result;
   },
+  async diagnosticNat() {
```

Comment:
```suggestion
  async runNatDiagnostic() {
```
use this format for all of these, including the IPC calls